contracts/src/v0.8/SimpleWriteAccessController.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./dev/ConfirmedOwner.sol";
+import "./interfaces/AccessControllerInterface.sol";
+
+/**
+ * @title SimpleWriteAccessController
+ * @notice Gives access to accounts explicitly added to an access list by the
+ * controller's owner.
+ * @dev does not make any special permissions for externally, see
+ * SimpleReadAccessController for that.
+ */
+contract SimpleWriteAccessController is AccessControllerInterface, ConfirmedOwner {
```

Comment:
`ConfirmedOwner` requires now a first owner on the constructor (https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/dev/ConfirmedOwner.sol#L22).
Should we give `SimpleWriteAccessController` the `msg.sender` as default owner? Or configurable from the constructor as well?

Reply:
Let's stick with `msg.sender` so it follows the same empty parameter set for the constructor as old versions of `SimpleWriteAccessController`.